platformio.ini

```diff
@@ -55,14 +55,16 @@ debug_tool = jlink
 
 debug_init_break = tbreak setup
 
+; Note: some libraries are specified by #ID where there are conflicting library
+; names.
 lib_deps =
   https://github.com/geeksville/RadioHead.git
-  TinyGPSPlus
+  1655 ; TinyGPSPlus
   https://github.com/geeksville/esp8266-oled-ssd1306.git ; ESP8266_SSD1306 
   AXP202X_Library
   SPI
-  OneButton
-  CRC32 ; explicitly needed because dependency is missing in the ble ota update lib
+  1260 ; OneButton
+  1202 ; CRC32, explicitly needed because dependency is missing in the ble ota update lib
```

Comment:
just curious - why did you need to use the numeric versions here?

Reply:
There are multiple libraries with the same name. If you don't specify one by number, it spits out these warnings:

![image](https://user-images.githubusercontent.com/260191/75612463-7a544400-5ad8-11ea-96c8-27845f058956.png)

(I wish they had a nicer syntax, e.g., `OneButton#1260` or something)